### DISCUSSION TITLE
Charmed deployment 1.8 stuck on postgres "awaiting for primary endpoint to be ready"

### DISCUSSION THREAD

**Discussion by kbaccar-core** [magma/magma#15447] [Q&A]
Hello, 
Im following the main documentation to run charmed magma. Everything went approximately as expected, except for the deployment of the charm with this command: _juju deploy magma-orc8r --overlay overlay.yaml --trust --channel=latest_
Apparently the postgres application is stuck on " awaiting for primary endpoint to be ready". 

<img width="635" alt="magmacore1" src="https://github.com/magma/magma/assets/174700204/849fe9c7-3e3b-40ed-b536-36ecc406d5ee">

Did anyone face this before ?

**Comment by lucasgonze** [comment]
I think the Ubuntu people who developed the charm would be the first ones to ask. Search Slack for ubuntu and charm.

**Comment by lucasgonze** [comment]
Guillaume Belanger from Canonical would be the expert. @ that name in Slack.

**Comment by lucasgonze** [comment]
I'll close this as stale unless you are still blocked.
